content/foundations/icons/octovisuals.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Octovisuals
+source: https://github.com/github/octovisuals
+---
+
+import {Box, Text, Link} from '@primer/react'
+import {LinkExternalIcon, DownloadIcon} from '@primer/octicons-react'
+import OctovisualsBanner from './images/octovisuals.png'
+import './octovisuals.css'
+
+### Octovisuals are GitHub's internal set of logos and visuals for brand and marketing usage.
+
+<br />
+
+GitHub's brand is visually expressed in a multitude of ways depending on the project, audience, and more. We use various brand assets, such as Octovisuals, to represent differnt aspects of GitHub. **Octovisuals are intended for internal use only and can only be accessed by Github staff.**
+
+<p>
+  <a href="https://octovisuals.github.com/" target="_blank">
+    See the full set of Octovisuals here
+  </a>
+  &nbsp;
+  <LinkExternalIcon size={16} fill="#0969da" />
+  &nbsp;&nbsp;<span style="color:#6E7781">(GitHub staff only)</span>
+</p>
+
+<br />
+
+<img src={OctovisualsBanner} style="width:100%" alt="Octovisuals" />
+
+<hr />
+
+### Permitted GitHub logos
+
+<br />
+
+<Box className="octovisual_assets">
+  <Box
+    sx={{
+      margin: '0px 24px 32px 24px',
+      border: '1px solid #d0d7de',
+      borderTopRightRadius: '3px',
+      borderTopLeftRadius: '3px',
+      borderBottomRightRadius: '6px',
+      borderBottomLeftRadius: '6px',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      flexDirection: 'column',
+    }}
+  >
+    <div
+      style={{
+        height: 200,
+        display: 'flex',
+        alignItems: 'center',
+      }}
+    >
+      <img
+        width="200"
+        height="52"
+        src="https://github.githubassets.com/images/modules/logos_page/GitHub-Logo.png"
+        alt="GitHub Logotype"
+        style="height: auto"
+      />
+    </div>
+    <a
+      style={{
+        borderTop: '1px solid #d0d7de',
+        backgroundColor: '#f6f8fa',
+        padding: '16px',
+        width: '100%',
+        textAlign: 'center',
+        textDecoration: 'none',
+      }}
+      href="https://github-media-downloads.s3.amazonaws.com/GitHub-Logos.zip"
```

Comment:
**Not related to this PR** 

I believe we shouldn't be hosting anything on AWS buckets anymore. cc. @raytalks @lesliecdubs in case we need to follow up on this separately.